configure.ac

```diff
@@ -23,6 +23,11 @@ define([prj_ver_release], esyscmd([echo ${PRJ_RELEASE:=0}]))
 #
 AC_INIT(libxlio, [prj_ver_major.prj_ver_minor.prj_ver_revision], support@mellanox.com)
 
+# Force sysconfdir to /etc regardless of prefix
+# e.g. - "xlio_config_schema.json" will always be placed at "/etc"
+sysconfdir="/etc"
+AC_SUBST([sysconfdir])
+
```

Comment:
Consider `configure --prefix=<path to install>`
w/o setting --sysconfdir explicitly it should install into `<path to install>/etc` and it is good.

Does not this update force moving files into <root>/etc ? I am not sure it is good approach.
Use `./configure --prefix=/your/custom/path --sysconfdir=/etc`

Reply:
i think you're right - we tried to solve a bug reported by verification but they honestly just need to run as you suggested.